tests/test_entities.py

```diff
@@ -1562,6 +1579,23 @@ def test_organization_rh_repo_url(self):
         ).update_payload()
         self.assertIn('redhat_repository_url', payload)
 
+    def test_os_default_template(self):
+        """Check, that os_default_template serves template_kind_id and
+        provisioning_template_id only wrapped in sub dict
+        See: http://projects.theforeman.org/issues/21169
```

Comment:
Same here - `/s/os_default_template/``os_default_template``/`, same for `template_kind_id` and `provisioning_template_id`.
And pls highlight the link 🙂